lib/extensions/rect.dart

```diff
@@ -21,6 +21,12 @@ class RectFactory {
     return Rect.fromLTWH(0, 0, size.width, size.height);
   }
 
+  /// Creates a [Rect] having its left upper corner in start and its right
+  /// bottom corner in end
+  static Rect fromVectors(Vector2 start, Vector2 end) {
+    return Rect.fromLTWH(start.x, start.y, end.x, end.y);
+  }
```

Comment:
this comment is **not** what the method does!!!
it's LFT**WH** not LF**RB** :)

also, I thought we had start.toRect(end) for this

Reply:
Very true, and the other one we already have `size.toRect()` for.
Removing both.